cinema/views.py

```diff
@@ -118,12 +156,18 @@ class OrderPagination(PageNumberPagination):
     max_page_size = 100
 
 
-class OrderViewSet(viewsets.ModelViewSet):
+class OrderViewSet(
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+    viewsets.GenericViewSet
+):
     queryset = Order.objects.prefetch_related(
         "tickets__movie_session__movie", "tickets__movie_session__cinema_hall"
     )
     serializer_class = OrderSerializer
     pagination_class = OrderPagination
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (IsAuthenticated,)
```

Comment:
The `OrderViewSet` uses `IsAuthenticated` for permission classes, which is inconsistent with other viewsets that use `IsAdminOrIfAuthenticatedReadOnly`. Ensure that this choice aligns with the application's requirements and adjust if necessary.